src/sentry/api/helpers/group_index/update.py

```diff
@@ -204,6 +211,8 @@ def update_groups(
         )
         if not serializer.is_valid():
             raise serializers.ValidationError(serializer.errors)
+        if not hasPriority and features.has("projects:issue-priority", project, actor=user):
```

Comment:
I have this check inside this loop for projects to get a project value. Once set, it should not have to check for the feature in subsequent loops.